schema/reimbursement_admin.sql

```diff
@@ -0,0 +1,45 @@
+-- phpMyAdmin SQL Dump
+-- version 4.8.3
+-- https://www.phpmyadmin.net/
+--
+-- Server version: 5.7.24-log
+-- PHP Version: 7.2.10
+
+SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
+SET AUTOCOMMIT = 0;
+START TRANSACTION;
+SET time_zone = "+00:00";
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8mb4 */;
+
+--
+-- Database: `tribunal`
+--
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `reimbursement_admin`
+--
+
+CREATE TABLE `reimbursement_admin` (
+  `admin_name` text NOT NULL,
+  `admin_email` text NOT NULL,
+  `super_email` text NOT NULL
```

Comment:
Use `varchar` instead of `text` as it's a little bit more performant. Plus, there will be an upper limit to how long an email can be.